uvicorn/loops/uvloop.py

```diff
@@ -4,6 +4,5 @@
 
 
 def uvloop_setup():
-    asyncio.get_event_loop().close()
```

Comment:
Not sure if this is actually needed. If I keep the line, it complains with `RuntimeError: Cannot close a running event loop` (using uvloop only).